nipype/interfaces/ants/tests/test_resampling.py

```diff
@@ -0,0 +1,93 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:                                                                                                                 
+
+from nipype.interfaces.ants import WarpImageMultiTransform, WarpTimeSeriesImageMultiTransform
+import os
+import pytest, pdb
+
+
+@pytest.fixture()
+def change_dir(request):
+   orig_dir = os.getcwd()
+   filepath = os.path.dirname( os.path.realpath( __file__ ) )
+   datadir = os.path.realpath(os.path.join(filepath, '../../../testing/data'))
+   os.chdir(datadir)
+
+   def move2orig():
+      os.chdir(orig_dir)
+
+   request.addfinalizer(move2orig)
+
+
+
+def test_WarpImageMultiTransform(change_dir):
+   wimt = WarpImageMultiTransform()
+   wimt.inputs.input_image = 'diffusion_weighted.nii'
+   wimt.inputs.reference_image = 'functional.nii'
+   wimt.inputs.transformation_series = ['func2anat_coreg_Affine.txt','func2anat_InverseWarp.nii.gz', \
+                                           'dwi2anat_Warp.nii.gz','dwi2anat_coreg_Affine.txt']
+   assert wimt.cmdline == 'WarpImageMultiTransform 3 diffusion_weighted.nii diffusion_weighted_wimt.nii -R functional.nii \
+func2anat_coreg_Affine.txt func2anat_InverseWarp.nii.gz dwi2anat_Warp.nii.gz dwi2anat_coreg_Affine.txt'
+
+
+def test_WarpImageMultiTransform_invaffine_1(change_dir):
+   wimt = WarpImageMultiTransform() 
+   wimt.inputs.input_image = 'diffusion_weighted.nii'
+   wimt.inputs.reference_image = 'functional.nii'
+   wimt.inputs.transformation_series = ['func2anat_coreg_Affine.txt','func2anat_InverseWarp.nii.gz', \
+                                           'dwi2anat_Warp.nii.gz','dwi2anat_coreg_Affine.txt']  
+   wimt.inputs.invert_affine = [1]
+   assert wimt.cmdline == 'WarpImageMultiTransform 3 diffusion_weighted.nii diffusion_weighted_wimt.nii -R functional.nii \
+-i func2anat_coreg_Affine.txt func2anat_InverseWarp.nii.gz dwi2anat_Warp.nii.gz dwi2anat_coreg_Affine.txt'
+
+
+def test_WarpImageMultiTransform_invaffine_2(change_dir):
+   wimt = WarpImageMultiTransform()
+   wimt.inputs.input_image = 'diffusion_weighted.nii'
+   wimt.inputs.reference_image = 'functional.nii'
+   wimt.inputs.transformation_series = ['func2anat_coreg_Affine.txt','func2anat_InverseWarp.nii.gz', \
+                                           'dwi2anat_Warp.nii.gz','dwi2anat_coreg_Affine.txt']
+   wimt.inputs.invert_affine = [2]
+   assert wimt.cmdline == 'WarpImageMultiTransform 3 diffusion_weighted.nii diffusion_weighted_wimt.nii -R functional.nii func2anat_coreg_Affine.txt func2anat_InverseWarp.nii.gz dwi2anat_Warp.nii.gz -i dwi2anat_coreg_Affine.txt'
+
+
+@pytest.mark.xfail(reason="dj: should it fail?")
```

Comment:
this should not fail.

Reply:
how does it fail?

Reply:
@satra - fails, i.e. it doesn't raise any exception.

I found it slightly confusing that i can provide wrong index in invert_affine, e.g. 0 or 3 (if you have only 2 affine files), and i don't get any errors. Was wondering if it would be useful to add an extra check.

Reply:
i think the issue is that the description is not very clear.

technically, looking at the code, each affine is given a index. seems like this can cause some mishaps. so yes, raising exceptions when the values of index exceed total number of affines is good.

Reply:
I think that description of `invert_affine` trait is quite good. However IMO it's very hard to guess without reading description or the code that affine files are counted, they are getting another indices, and the new indices start from 1. 

Unfortunately changing this concept would make old script fail, so was thinking that at least adding some error with meaningful message might save people some time in case of problems.

Reply:
that sounds good, but i think it may be good to add an example to describe this situation clearly in the docstring examples.

Reply:
I add a comment to an existing doctest and add a new one, please see if the new one really returns cmd that makes sense